build.rs

```diff
@@ -32,7 +32,10 @@ fn main() -> Result<()> {
     tonic_build::configure()
         .build_server(true)
         .build_client(true)
-        .type_attribute(".", "#[derive(serde_derive::Serialize)]")
+        .type_attribute(
+            ".",
+            "#[derive(serde_derive::Serialize, serde_derive::Deserialize)]",
+        )
         .type_attribute(".helium.config", "#[derive(serde_derive::Deserialize)]")
```

Comment:
Remove the `.helium.config` attribute since the `.` pattern match in this PR adds it to all protos